auth/src/main/java/feast/auth/config/SecurityConfig.java

```diff
@@ -89,7 +89,7 @@ GrpcAuthenticationReader authenticationReader() {
    * @return AccessDecisionManager
    */
   @Bean
-  @ConditionalOnProperty(prefix = "feast.security.authorization", name = "enabled")
+  @ConditionalOnProperty(prefix = "feast.security.authentication", name = "enabled")
```

Comment:
We might have to revert this change till #865 gets merged in. As you already know, If any deployment has authentication enabled but not authorization, serving boot up will fail due to connection dependency on core.

Reply:
Correct.